script/nohup_wrapper.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+# TODO redirect to sane location
+nohup $* > /tmp/aion-kernel-nohup &
```

Comment:
this should be placed in the local folder, otherwise, the user may never become aware of it

Reply:
Ok. That sounds good.